frame/lending/src/lib.rs

```diff
@@ -94,32 +97,51 @@ pub mod pallet {
 		offchain::{AppCrypto, CreateSignedTransaction, SendSignedTransaction, Signer},
 		pallet_prelude::*,
 	};
-	use num_traits::CheckedDiv;
 	use sp_core::crypto::KeyTypeId;
 	use sp_runtime::{
-		traits::{AccountIdConversion, CheckedAdd, CheckedMul, CheckedSub, One, Saturating, Zero},
+		traits::{AccountIdConversion, One, Saturating, Zero},
 		ArithmeticError, DispatchError, FixedPointNumber, FixedU128, KeyTypeId as CryptoKeyTypeId,
 		Percent, Perquintill,
 	};
 	use sp_std::{fmt::Debug, vec, vec::Vec};
 
-	type MarketConfiguration<T> = MarketConfig<
+	/////////////////////////////////////////////////////////////////////////
+	// Definitions
+	/////////////////////////////////////////////////////////////////////////
+
+	/// Simple type alias around [`MarketConfig`] for this pallet.
+	type MarketConfigOf<T> = MarketConfig<
 		<T as Config>::VaultId,
 		<T as DeFiComposableConfig>::MayBeAssetId,
 		<T as frame_system::Config>::AccountId,
 		<T as Config>::LiquidationStrategyId,
 	>;
 
+	// REVIEW: Maybe move this to `models::market_index`?
 	#[derive(Default, Debug, Copy, Clone, Encode, Decode, PartialEq, MaxEncodedLen, TypeInfo)]
 	#[repr(transparent)]
-	pub struct MarketIndex(u32);
+	pub struct MarketIndex(
+		#[cfg(test)] // to allow pattern matching in tests outside of crate
+		pub u32,
+		#[cfg(not(test))] pub(crate) u32,
+	);
 
+	// REVIEW: Should this be `Id`? Index is confusing.
```

Comment:
yep, id is better, can leave todo. and overall `type MarketId` is better than wrapper. no need to make it now.;